README.md

```diff
@@ -1,13 +1,36 @@
 [![Type Hero Logo](/logo.svg)](https://typehero.dev/)
 
+<!-- do we want badge placeholders?? -->
+
 # TypeHero
 
-Connect, collaborate, and grow with a community of TypeScript developers. Elevate your skills through interactive coding challenges, discussions, and knowledge sharing
+Connect, collaborate, and grow with a community of TypeScript developers.
+Elevate your skills through interactive coding challenges, discussions, and
+knowledge sharing
+
+## About Us
+
+TypeScript's typing system can be very powerful when used correctly. The problem
+is that most developers or engineers don't use much more than they have to in
+their day to day. Inevitably a moment comes where their knowledge just doesn't
+cut it for a use case, and they have to reach out to a community for help. We
+aim to be not only the community but the platform they can use to learn more of
+the intricacies of TypeScript's typing system.
 
 ## Quick Start
 
-Please see the [`CONTRIBUTING.md`](/CONTRIBUTING.md)
+Please see the [`LOCAL.md`](/LOCAL.md)
 
 ## Tech Stack
 
-Please see the [`package.json`](/package.json)
+Our tech stack consists of many technologies, primarily
```

Comment:
id prefer not to list out all the deps. its just a laundry list

Reply:
what if we simplify it to just 

1. TypeScript
2. NextJs
3. Tailwind
4. Vercel (our gracious sponser)

and then go "plus many more you can see in the `package.json` file"

Reply:
Yeah I'd say omit the tech stack from the readme.

@bautistaaa you can add it to the repo description and say "built with nextjs".